cli/src/index.js

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
+
+const execSync = require("child_process").execSync;
+const meow = require("meow");
```

Comment:
Could we standardise around a CLI package? The jekyll-extract package in this repo uses commander, so it would be good to either change this package to use commander, or change the jekyll-extract package to use meow

Reply:
Ok, sweet, will do. Will change to commander

Reply:
Ok @bglw put commander in place of meow